components/places/src/types.rs

```diff
@@ -15,16 +15,17 @@ use std::time::{Duration, SystemTime, UNIX_EPOCH};
 pub struct SyncGuid(pub String);
 
 impl SyncGuid {
+    #[allow(clippy::new_without_default)] // This probably should not be called `new`...
```

Comment:
Ehhh. I see where it's going with that, but implementing `Default` doesn't really make sense here, and `generate` is a mouthful. Sometimes Clippy is overeager.

Reply:
I think renaming it SyncGuid::rand() would be reasonable